R/J.R

```diff
@@ -0,0 +1,29 @@
+#############################################################################
+#
+# XLConnect
+# Copyright (C) 2010-2021 Mirai Solutions GmbH
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+#############################################################################
+
+#############################################################################
+#
+# Override rJava's J() to use the dedicated classloader, as advised by
+# Simon Urbanek in the rJava CHANGELOG (https://cran.r-project.org/package=rJava/NEWS)
+# 
+# Author: Simon Poltier, Mirai Solutions GmbH
+#
+#############################################################################
+J <- function(...) rJava::J(..., class.loader=.rJava.class.loader)
```

Comment:
Not sure if I should include this in a different way, since it is copied straight from the NEWS file.